HelloWorld/HelloWorld/ViewController.swift

```diff
@@ -23,7 +23,7 @@ class ViewController: UIViewController {
 
 
     @IBAction func buttonDidClick(_ sender: Any) {
-        helloLabel.text = "Super Hello, world. TEST!"
+        helloLabel.text = "Super Hello, world."
```

Comment:
Because it's the requirement from my manager. She thinks, TEST here is kinda redundancy.